### DISCUSSION TITLE
Synchronized external links

### DISCUSSION THREAD

**Discussion by bangetto** [modrinth/.github#50] [Feature requests]
Imagine, you have already created a lot of projects and you create yourself a website, or open a new donation method or a Discord server, you have to go to every single project and ad it, kind of frustrating, and if not even all of them, because an issue tracker is diferent for every project, but something like a Discord server is usually one for all of the projects, but this could be an optional feature.

**Comment by triphora** [comment]
Besides for donation links (see modrinth/knossos#851) the ability already exists to bulk edit external project links. Go to your dashboard and then to projects. You can check all the projects you want to edit the links of and then enter whatever you want.
